container_files/public_html/doc/builtin/sql/ValueExpression.md

```diff
@@ -387,6 +387,7 @@ expression|result
 - `replace_inf(x, y)`: replace all `Inf`s and `-Inf`s in `x` by `y`.
 - `binomial_lb_80(trials, successes)` returns the 80% lower bound using the Wilson score.
 - `binomial_ub_80(trials, successes)` returns the 80% upper bound using the Wilson score.
+- `clamp(x,lower,upper)` will clamp the value 'x' between the lower and upper bounds.
```

Comment:
We can be more explicit about what the function does.  I think you gave examples in the previous version of that PR.